nautobot/extras/models/models.py

```diff
@@ -184,8 +184,15 @@ def get_config_context(self):
         Return the rendered configuration context for a device or VM.
         """
 
-        # always manually query for config contexts
-        config_context_data = ConfigContext.objects.get_for_object(self).values_list("data", flat=True)
+        if not hasattr(self, "config_context_data"):
+            # Annotation not available, so fall back to manually querying for the config context
+            config_context_data = ConfigContext.objects.get_for_object(self).values_list("data", flat=True)
+        else:
+            config_context_data = self.config_context_data or []
```

Comment:
I'm curious as to what the or is doing here. The `hasattr` above should mean this is always truthy. Is it more desirable to do a type check here?